Fix AQL grammar to support 'as alias' in select

pieterbos opened the pull request:
Fixes https://github.com/nedap/archetype-languageserver/issues/26

EBrader commented on vscode-extension/syntaxes/aql.tmGrammar.json:
```diff
@@ -35,8 +35,10 @@
 				{ "include": "#number"  },
+				{ "include": "#as_name"  },
 				{ "include": "#keywords"  },
 				{ "include": "#path_segment"  },
+
 				{ "include": "#contains"  }
```
```suggestion
				{ "include": "#path_segment"  },
				{ "include": "#contains"  }
```

EBrader reviewed (approved):
If the AS should... 
- Never start with a number and only a-zA-Z 
- And not be followed by a pattern containing `-`
... then this is all good 👍🏻

pieterbos commented:
Yes, it starts with only a-zA-Z, then followed by a-zA-Z, numeric characters or an underscore. 

I added a new commit, since the last one only accepted space characters, and not other whitespace characters. It should, so I added those to the grammar. Could you review again?
